conductor_api/Cargo.toml

```diff
@@ -35,13 +35,12 @@ maplit = "=1.0.1"
 lazy_static = "=1.2.0"
 json-patch = "=0.2.2"
 hyper = "=0.12.25"
```

Comment:
Do we need hyper at all now?
```suggestion
```

Reply:
It looks like it's still being used for http::uri and for Request